src/main/java/org/apache/commons/csv/CSVParser.java

```diff
@@ -734,9 +734,16 @@ private boolean isStrictQuoteMode() {
      * {@link IllegalStateException}.
      * </p>
      * <p>
-     * If the parser is closed a call to {@link Iterator#next()} will throw a
+     * If the parser is closed, the iterator will not yield any more records.
+     * A call to {@link Iterator#hasNext()} will return {@code false} and
+     * a call to {@link Iterator#next()} will throw a
      * {@link NoSuchElementException}.
      * </p>
+     * <p>
+     * If it is necessary to construct an iterator which is usable after the
+     * parser is closed, one option is to extract all records as a list with
+     * {@link getRecords()}, and return an iterator to that list.
```

Comment:
@pedro-w For future reference: `{@link getRecords()}` is NOT legal Javadoc syntax, use `{@link #getRecords()}`

Reply:
Sorry, I didn't realise that (also I thought I'd addressed all the warnings displayed when building the project). I see you fixed it in 22df0e28c470e98b3eee74420d53dc686be4329c, thanks!